package.json

```diff
@@ -35,8 +35,8 @@
     "history": "^5.0.0",
     "moment": "^2.27.0",
     "react": "^16.13.1",
-    "react-router": "6.0.0-beta.0",
     "react-dom": "^16.13.1",
+    "react-router": "6.0.0-beta.0",
     "react-use": "^15.3.3"
```

Comment:
@iain-b I'm confused by the diff. I was expecting @backstage/xyz dependencies to be bumped:

```
  "dependencies": {
    "@backstage/catalog-model": "^0.1.1-alpha.22",
    "@backstage/core": "^0.1.1-alpha.22",
    "@backstage/plugin-catalog": "^0.1.1-alpha.22",
    "@backstage/theme": "^0.1.1-alpha.22",
```